model/ExpandedLicensing/Classes/IndividualLicense.md

```diff
@@ -0,0 +1,21 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# IndividualLicense
+
+## Summary
+
+A concrete subclass of AnyLicenseInfo used by Individuals in the ExpandedLicensing profile.
```

Comment:
See my overall comment about whether to use the approach described here or the alternative you mentioned in the discussion thread. Assuming we use the one you did here, though, then is it accurate that IndividualLicense is a "concrete" subclass? Or is it an abstract subclass, since only the NoneLicense and NoAssertionLicense classes should actually be instantiated?

Reply:
Thanks @swinslow - really good catches.

After reviewing the comments, I no longer see a need for the IndividualLicense class, I can just subclass AnyLicenseInfo directly for None and NoAssertion license types.

I'll update the PR accordingly.